### ISSUE TITLE
I was unable to uninstall IRkernel.

### ISSUE THREAD

**Issue by ghost** [IRkernel/IRkernel#434]
I am using Mac OS and I have r installed. I used terminal to install IRkernel and this message keeps repeating when I try to do this.

```r
devtools::install_github('IRkernel/IRkernel')
```

Result:

```
Downloading GitHub repo IRkernel/IRkernel@master
from URL https://api.github.com/repos/IRkernel/IRkernel/zipball/master
Installing IRkernel
'/Library/Frameworks/R.framework/Resources/bin/R' --no-site-file --no-environ  \
  --no-save --no-restore --quiet CMD INSTALL  \
  '/private/var/folders/j_/dqs9qh055599_0qb22lmjs0w0000gn/T/RtmpfTsQBR/devtools71e5e138c8a/IRkernel-IRkernel-ff78a04'  \
  --library='/Library/Frameworks/R.framework/Versions/3.3/Resources/library'  \
  --install-tests 

During startup - Warning messages:
1: Setting LC_CTYPE failed, using "C" 
2: Setting LC_TIME failed, using "C" 
3: Setting LC_MESSAGES failed, using "C" 
4: Setting LC_MONETARY failed, using "C" 
* installing *source* package 'IRkernel' ...
** R
** inst
** tests
** preparing package for lazy loading
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
Warning: running command '/usr/bin/otool -L /Library/Frameworks/R.framework/Versions/3.3/Resources/library/pbdZMQ/libs/pbdZMQ.so' had status 1
Error : .onLoad failed in loadNamespace() for 'pbdZMQ', details:
  call: if (org != fn.libzmq.4.dylib) {
  error: missing value where TRUE/FALSE needed
ERROR: lazy loading failed for package 'IRkernel'
* removing '/Library/Frameworks/R.framework/Versions/3.3/Resources/library/IRkernel'
Error: Command failed (1)
```

Does anyone know how to fix this problem?

**Comment by takluyver** [comment]
This seems to be the place where it goes wrong:

```
error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
```

That sounds Mac-related; I don't know what it's looking for or why it's not finding it.

**Comment by vikjam** [comment]
Hi @jspark971, you may find the instruction here useful: http://www.michaelpacer.com/maths/r-kernel-for-ipython-notebook

In particular, see the part about ZMQ.

**Comment by flying-sheep** [comment]
fix in #490
